pom.xml

```diff
@@ -84,8 +84,8 @@
     <properties>
         <jakarta.el.version>5.0.0</jakarta.el.version>
         <cdi-api.version>4.0.1</cdi-api.version>
-        <cdi.tck.version>4.0.6</cdi.tck.version>
-        <weld.version>5.0.1.Final</weld.version>
+        <cdi.tck.version>4.0.13</cdi.tck.version>
+        <weld.version>5.1.2.Final</weld.version>
```

Comment:
I'm not sure we should be updating Weld that much, we purposefully use Weld 5.0.x in Payara due to breaking changes.